src/components/Grunnlag/Grunnlag.tsx

```diff
@@ -29,27 +28,20 @@ const useUttaksgrad = (): number => {
 
 interface Props {
   tidligstMuligUttak: Uttaksalder
+  pensjonsavtaler: Pensjonsavtale[]
+  showLoader: boolean
+  showError: boolean
 }
 
-export function Grunnlag({ tidligstMuligUttak }: Props) {
+export function Grunnlag({
+  tidligstMuligUttak,
+  pensjonsavtaler,
+  showLoader,
+  showError,
```

Comment:
Det blir fort rotete om `Grunnlag` skal ta imot props som sier om vi skal vise loader/error for all dataen som skal hentes og vises her. Det er som regel en god ide å flytte dette nærmere der det brukes, og i dette tilfellet burde ikke disse tas inn som props i `Grunnlag` men være noe som `Pensjonsavtaler` håndterer internt i komponenten

Reply:
Ref svar over.